server/controllers/songsController.js

```diff
@@ -1,39 +1,49 @@
 import asyncHandler from "../middleware/asyncHandler.js";
 import Song from "../models/Song.js";
+import logger from "../logger.js";
 import ErrorResponse from "../utils/ErrorResponse.js";
 import createObjectFromQuery from "../utils/createObjectFromQuery.js";
 import { getOrCreateArtist } from "./artistsController.js";
 import { createDummySong } from "../utils/createDummyData.js";
 
 const findSong = async (req) => {
   const filter = createObjectFromQuery(req.body);
+  logger.info(`findSong with song details ${filter} sent`);
```

Comment:
you did not send it yet. this might confuse us when we debug, because we will think that this log happens after line 12